app/phone/src/main/java/dev/jdtech/jellyfin/PlayerActivity.kt

```diff
@@ -69,10 +75,21 @@ class PlayerActivity : BasePlayerActivity() {
             videoNameTextView.text = title
         }
 
+        binding.playerView.addOnLayoutChangeListener { v: View?, oldLeft: Int,
+                                                       oldTop: Int, oldRight: Int, oldBottom: Int, newLeft: Int, newTop:
+                                                       Int, newRight: Int, newBottom: Int ->
+            if (viewModel.player is MPVPlayer) {
+                binding.playerView.getGlobalVisibleRect(sourceRectHint)
+            } else {
+                binding.playerView.videoSurfaceView?.getGlobalVisibleRect(sourceRectHint)
+            }
+        }
+
```

Comment:
Does this actually do anything?

Reply:
Yes, it's used for a smooth transition when using the pip button or in android 12+ (not implemented) when `.setAutoEnterEnable(true)`.

Reply:
MPV seems to work with `binding.playerView.videoSurfaceView?.getGlobalVisibleRect(sourceRectHint)` so I removed the ifelse

Reply:
Works only with exoplayer, with MPV I wasn't able to get the `videoSurfaceView`